### DISCUSSION TITLE
Downloaded mumps could not be used.

### DISCUSSION THREAD

**Discussion by AnthonyB08** [neams-th-coe/cardinal#589] [Q&A] [answered]
Hi all,

I am wondering if anyone has come across this error while running the command :./contrib/moose/scripts/update_and_rebuild_petsc.sh 

![image](https://user-images.githubusercontent.com/55299169/218544857-92670073-213e-4dec-b5b4-8adf94d23188.png)

**Comment by aprilnovak** [comment]
Hi @AnthonyB08 - I'm not sure on this one. Would you please re-post it in the [MOOSE discussions page](https://github.com/idaholab/moose/discussions)? For PETSc and libMesh issues, they'll probably be able to provide more assistance.

**Comment by AnthonyB08** [comment, marked as answer]
For future and new users, i.e fresh install of ubuntu. Refer to this discussion as it fixed all of my issues and I am now able to run cardinal successfully. 

https://github.com/idaholab/moose/discussions/23408#discussioncomment-4974604
